tools/jit/gen_jit_dispatch.py

```diff
@@ -147,7 +147,15 @@ def from_ivalue(arg, value):
 
 CONSTRUCTOR = CodeTemplate("""\
 [](Stack & stack) {
-    autograd::profiler::RecordFunction record("${name}");
+    autograd::profiler::RecordFunction record("${name}", [&stack]() {
```

Comment:
This is unnecessary. Autogenerated functions like this one will almost immediately dispatch to eager versions of those ops, triggering another `RecordFunction`.

Reply:
In fact the entire call to RecordFunction here can be deleted.